.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+documentation:
+  - name: 📘 Documentation
+    url: https://mongoosejs.com/docs/guide.html
+    about: The manual of mongoose
```

Comment:
"The manual of mongoose" sounds a bit strange. Let's just put "Mongoose docs"

Reply:
done